keyboards/boardsource/equals/48/info.json

```
@@ -0,0 +1,176 @@
+{
+    "keyboard_name": "Equals 48",
+        "bootloader": "rp2040",
+    "processor": "RP2040",
+    "features": {
+        "audio":true
+    },
+    "matrix_pins": {
+        "cols": ["GP0", "GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP10", "GP11"],
+        "rows": ["GP12", "GP13", "GP16", "GP17"]
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
```

This has been renamed, and needs to be updated: 
```suggestion
        "driver": "ws2812",
```